user-org-util/pom.xml

```diff
@@ -49,5 +49,17 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
think this (everit schema) was unrelated change to this PR. remove it from this PR.